simularium/binding-sim-edu
# Feature/resize hook

Problem
=======
Estimated review size: small
All of the panels need to know when the screen has changed sizes 

closes #62 

Solution
========
created a hook and a component that gets a new size when the window changes 
Still to do: resize the 'Record' button 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. bun dev
2. get to simulation 
3. press play to get the plots to show 
4. they should be sized well
5. resize the window and the plots should re-render at the correct size again 

Screenshots (optional):
-----------------------
<img width="1544" alt="Screenshot 2024-04-30 at 2 15 21 PM" src="https://github.com/simularium/binding-sim-edu/assets/5170636/a7b55bc7-038e-4924-8b5d-4b820af52441">
